lib/govuk_tech_docs.rb

```diff
@@ -40,15 +43,19 @@ def self.configure(context, options = {})
     context.files.watch :source, path: "#{__dir__}/source"
 
     context.set :markdown_engine, :redcarpet
-    context.set :markdown,
-                renderer: TechDocsHTMLRenderer.new(
-                  with_toc_data: true,
-                  api: true,
-                  context: context,
-                ),
-                fenced_code_blocks: true,
-                tables: true,
-                no_intra_emphasis: true
+
+    default_markdown_options = {
+      renderer: TechDocsHTMLRenderer.new(
+        with_toc_data: true,
+        api: true,
+        context: context,
+      ),
+      fenced_code_blocks: true,
+      tables: true,
+      no_intra_emphasis: true
+    }
+    markdown_options = context.config[:tech_docs][:markdown].transform_keys(&:to_sym) || {}
```

Comment:
What happens if there are no markdown options specified in the config file? Does this need to be guarded?

Reply:
I believe It'll return falsy and the {} will be used.

Reply:
Doesn't seem to like the chaining when I try it:

```
irb(main):004:0> foo = { :tech_docs => { :something => true }}
=> {:tech_docs=>{:something=>true}}
irb(main):005:0> foo[:tech_docs][:markdown].transform_keys(&:to_sym) || {}
Traceback (most recent call last):
        4: from /Users/oliverbyford/.rbenv/versions/2.6.3/bin/irb:23:in `<main>'
        3: from /Users/oliverbyford/.rbenv/versions/2.6.3/bin/irb:23:in `load'
        2: from /Users/oliverbyford/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
        1: from (irb):5
NoMethodError (undefined method `transform_keys' for nil:NilClass)
```

Looks like it works OK with safe navigation (`&.`) though:

```
irb(main):004:0> foo = { :tech_docs => { :something => true }}
=> {:tech_docs=>{:something=>true}}
irb(main):006:0> foo[:tech_docs][:markdown]&.transform_keys(&:to_sym) || {}
=> {}
irb(main):007:0> foo = { :tech_docs => { :something => true, :markdown => { :be_awesome => true }}}
=> {:tech_docs=>{:something=>true, :markdown=>{:be_awesome=>true}}}
irb(main):008:0> foo[:tech_docs][:markdown]&.transform_keys(&:to_sym) || {}
=> {:be_awesome=>true}
```